src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -314,6 +314,29 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    folioPartnerArtistsAllLoader: gravityLoader(
```

Comment:
This can just be `partnerArtistsAllLoader`; another app might want to come along and use the partner loader for something else

Reply:
yeah the only reason I had the prefix was because in the unauthenticated loaders there is another loader that has the same name but hits a different endpoint which I found confusing: https://github.com/artsy/metaphysics/blob/483f4fdd95f8b5efc5e089b1a2488df880999b68/src/lib/loaders/loaders_without_authentication/gravity.ts#L117
but it is a bad name and was meant to be a placeholder